parser-typechecker/src/Unison/Codebase/BranchUtil.hs

```diff
@@ -60,12 +60,16 @@ getTerm (p, hq) b = case hq of
   filter sh = Set.filter (SH.isPrefixOf sh . Referent.toShortHash)
   terms = Branch._terms (Branch.getAt0 p b)
 
-getTermMetadataHQNamed :: (Path.Path, HQSegment) -> Branch0 m -> Metadata.R4 Referent NameSegment
+getTermMetadataHQNamed
+  :: (Path.Path, HQ'.HQSegment) -> Branch0 m -> Metadata.R4 Referent NameSegment
 getTermMetadataHQNamed (path, hqseg) b =
   R4.filter (\(r,n,_t,_v) -> HQ'.matchesNamedReferent n r hqseg) terms
   where terms = Metadata.starToR4 . Branch._terms $ Branch.getAt0 path b
 
-getTypeMetadataHQNamed :: (Path.Path, HQSegment) -> Branch0 m -> Metadata.R4 Reference NameSegment
+getTypeMetadataHQNamed
+  :: (Path.Path, HQ'.HQSegment)
+  -> Branch0 m
+  -> Metadata.R4 Reference NameSegment
```

Comment:
Formatting change only. Ended up reverting a change to these, but left the formatting change since it's nicer to read.